.github/workflows/tests.yml

```diff
@@ -55,12 +55,19 @@ jobs:
         with:
           username: locustbuild
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-      - uses: docker/build-push-action@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: build and publish image
+        id: docker_build
+        uses: docker/build-push-action@v2
+        if: ${{ github.ref == 'refs/heads/master' && github.repository_owner == 'locustio' }}
```

Comment:
The reason for having this if statement for the push-setting rather than the whole step is that we can see any errors *before* merging. Is it necessary to move it?